bindings/src/types.rs

```diff
@@ -357,6 +357,7 @@ pub struct ValidatorDetail {
 pub struct PoolAsset {
     pub token: Coin,
     pub weight: Uint128,
+    pub usd_value: Option<Uint128>,
```

Comment:
I believe this should be `Option<Decimal>` instead. Could you please check how it's being done in other queries?